src/cli/commands/__tests__/lint.test.ts

```diff
@@ -202,8 +202,7 @@ describe('lint', () => {
         .command(['lint', invalidOas3SpecPath, '-r', standardOas3RulesetPath])
         .it('outputs warnings in default format', ctx => {
           expect(ctx.stdout).toContain(
-            'warning  api-servers       OpenAPI `servers` must be present and non-empty array',
-            // Restore to once Windows new line is sorted '1:5  warning  api-servers       OpenAPI `servers` must be present and non-empty array',
+            '1:1  warning  api-servers       OpenAPI `servers` must be present and non-empty array',
```

Comment:
1:1? Wasn't it 1:5?

Reply:
![image](https://user-images.githubusercontent.com/9273484/60835558-c2c69680-a1c3-11e9-8f48-0e5688941095.png)
:laughing:

Reply:
TFW you try to add a description and he doesn't read it anyway :trollface:

Reply:
Yeah the template is so misleading that seems like nobody uses it and I just skip it.